allennlp/tests/commands/fine_tune_test.py

```diff
@@ -55,6 +57,50 @@ def test_fine_tune_works_with_vocab_expansion(self):
         assert tuple(extended_weight.shape) == (25, 300)
         assert torch.all(original_weight == extended_weight[:24, :])
 
+    def test_fine_tune_works_with_vocab_expansion_with_pretrained_file(self):
+        params = Params.from_file(self.config_file)
+        # snli2 has a new token (seahorse) in it
+        params["train_data_path"] = str(self.FIXTURES_ROOT / 'data' / 'snli2.jsonl')
+
+        embeddings_filename = str(self.TEST_DIR / "embeddings2.gz")
+        extra_token_vector = np.random.rand(300)
+        with gzip.open(embeddings_filename, 'wb') as embeddings_file:
+            extra_token_vector_str = " ".join(map(str, list(extra_token_vector)))
+            embeddings_file.write("seahorse {}\n".format(extra_token_vector_str).encode('utf-8'))
```

Comment:
why not just make this a test fixture rather than writing it out every time

Reply:
also, I don't know if we have official guidance on this, but prefer f-strings to str.format

Reply:
done. 👍